tools/pmunts_muntsos/config-pin.c

```diff
@@ -33,7 +33,13 @@
 #include <sys/param.h>
 
 #define NAMES		"/sys/firmware/devicetree/base/ocp/%s_pinmux/pinctrl-names"
-#define STATE		"/sys/devices/platform/ocp/ocp:%s_pinmux/state"
+
+#if board_soc == AM572X
```

Comment:
This need to be a run-time setting, not compile-time. The binary could be built and distributed as part of the distro.

Reply:
Ohh okay, I'll try to fix that.